gradio-blocks/README.md

```diff
@@ -0,0 +1,100 @@
+
+**Event Plan**
+
+main components of the event consist of:
+
+- Learning about Gradio and the new Blocks Feature
+- How Gradio and Hugging Face Spaces work together
+- Understanding the Hugging Face ecosystem
+- Share Demos in a Hugging Face organization for the Blocks event
+
+
+**Example spaces using Blocks**
+- [https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks](https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks)
+- [https://huggingface.co/spaces/merve/gr-blocks](https://huggingface.co/spaces/merve/gr-blocks)
+- [https://huggingface.co/spaces/osanseviero/tortoisse-tts](https://huggingface.co/spaces/osanseviero/tortoisse-tts)
+- [https://huggingface.co/spaces/akhaliq/CaptchaCracker](https://huggingface.co/spaces/akhaliq/CaptchaCracker)
+
+
+To participate in the event
+
+- Join the organization for Blocks event
+    - [https://huggingface.co/Gradio-Blocks](https://huggingface.co/Gradio-Blocks)
+- Join the discord
+    - [discord](https://discord.com/invite/feTf9x3ZSB)
+
+
+Participants will have a month to build and share Gradio demos using the Blocks feature. We will share a list of ideas of spaces that can be created using blocks or participants are free to try out their own ideas. At the end of the event, spaces will be evaluated for creativity and prizes will be given 
+
+
+potential ideas for creating spaces:
+
+
+- papers from https://paperswithcode.com/
+- themed spaces: Vision, NLP, Audio based spaces
+- Models from huggingface model hub
+- Models from other model hubs
+    - Tensorflow Hub: see example Gradio demos at https://huggingface.co/tensorflow
+    - Pytorch Hub: see example Gradio demos at https://huggingface.co/pytorch
+    - ONNX model Hub: see example Gradio demos at https://huggingface.co/onnx
+    - PaddlePaddle Model Hub: see example Gradio demos at https://huggingface.co/PaddlePaddle
+- participant ideas
+
+
+**Prizes (tentative list)**
+
+- HF PRO 1 month or 1 year!
+- 1st place 12 months, 2nd 6m, 3rd 3m, 4th 1m HF PRO.
+- Legendary prize, Lifetime HF PRO!!!
+- HF PRO badge on HF
+- Gradio badge on HF
+- Special event badge on HF
+- A lot more swag: t-shirts, hoodies, mugs
+- Embed the demo or highlight it
+- Time with someone on the Gradio team (unique ?)
+
+
+**Creating a Gradio demo on Hugging Face Spaces**
+
+Once a model has been picked from the choices above, you can share a model in a Space using Gradio
+
+Read more about how to add Gradio spaces [here](https://huggingface.co/blog/gradio-spaces).
+ 
+Steps to add Gradio Spaces to the Gradio Blocks Party org
+- Create a account on Hugging Face
+- Join the Gradio Blocks Party Organization by clicking Join Organization Button in the Org Page
+- Once approved add your space using the Gradio SDK
```

Comment:
```suggestion
3. Once your request is approved, add your space using the Gradio SDK
```

Also it would be good to point out that people can carry already existing demos from their own profile to Gradio